ruy/profiler/test.cc

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 #include <random>
 #include <thread>
 
-#include "testing/base/public/gunit.h"
+#include "gtest/gtest.h"
```

Comment:
Thanks for catching this and the suggested fix. I will need to fix this in the Google internal repository, in our copybara setup, in such a way that we can keep the google-internal path internally, while exporting something that works on github.

For that reason, it would be convenient if you could split the googletest-related changes out of this PR, so I take it already.  That includes also the WORKSPACE changes.

Reply:
nevermind, now that i see what needs to be done on our side, it's best for me to import your PR and make fixups on top.